arch/arm64/core/cache.c

```diff
@@ -139,6 +139,9 @@ int arm64_dcache_range(void *addr, size_t size, int op)
  */
 int arm64_dcache_all(int op)
 {
+#ifdef CONFIG_DT_HAS_XEN_XEN_ENABLED
```

Comment:
You should revert the workaround patch. In a separate commit.

Reply:
done